<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>435</width>
    <height>329</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="GroupBoxInfo" name="groupBox_info"/>
    </item>
    <item>
     <widget class="GraphicsWatchArea" name="graphicsView">
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>435</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>&amp;Файл</string>
    </property>
    <addaction name="action_newWatcher"/>
    <addaction name="action_Load"/>
    <addaction name="action_Save"/>
    <addaction name="action_Exit"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>&amp;Трекинг</string>
    </property>
    <addaction name="action_StartStop"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="action_StartStop">
   <property name="text">
    <string>Начать</string>
   </property>
  </action>
  <action name="action_newWatcher">
   <property name="text">
    <string>Новое</string>
   </property>
  </action>
  <action name="action_Exit">
   <property name="text">
    <string>Выход</string>
   </property>
  </action>
  <action name="action_Save">
   <property name="text">
    <string>Сохранить...</string>
   </property>
  </action>
  <action name="action_Load">
   <property name="text">
    <string>Загрузить</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>GroupBoxInfo</class>
   <extends>QGroupBox</extends>
   <header location="global">groupboxinfo.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>GraphicsWatchArea</class>
   <extends>QGraphicsView</extends>
   <header location="global">graphicswatcharea.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>action_Exit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>217</x>
     <y>164</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
